### DISCUSSION TITLE
ipxe configuration

### DISCUSSION THREAD

**Discussion by sasi202** [ipxe/ipxe#672] [Q&A]
hello, I'll be glad for some help. 
how do i change the ip for the ipxe http to walk through the server ip and not from the router gateway ip?
instead 192.168.1.1 i need to change it to 192.168.1.41. i cant boot from ipxe because of that.

![image](https://user-images.githubusercontent.com/96876541/177196297-6add888e-c5da-46fa-80c8-50b187cbac10.png)

**Comment by NiKiZe** [comment]
Are you using an embedded script, or are you using DHCP to send filename.
I would start to check DHCP config to find the filename used for iPXE boot, it could be as easy as to modify that.

If this iPXE binary or setup has been provided by someone else, then you might want to check with them.

**Comment by stappersg** [comment]
![afbeelding](https://user-images.githubusercontent.com/35434094/183633698-cdf15845-b1b6-43e5-ae5d-214c9c9bfa06.png)

@sasi202  What about a follow-up response?
